source/develop/release-management/features/storage/vm-leases.html.md

```diff
@@ -0,0 +1,292 @@
+---
+title: VM Leases
+category: feature
+authors: nsoffer
+feature_name: VM Leases
+feature_modules: engine,vdsm
+feature_status: Design
+---
+
+# VM Leases
+
+
+## Overview
+
+This feature adds the ability to acquire a lease per VM on storage
+without attaching the lease to a disk. Having a VM lease, we gain two
+important capabilities; avoiding split-brain, and starting a VM on
+another host if the original host becomes unreachable.  The later
```

Comment:
The engine's term is Non Responsive. Perhaps it should be mentioned (either in addition or instead of "unreachable")?

Also, you have a redundant space before "The".